src/main/java/org/broadinstitute/hellbender/engine/FeatureManager.java

```diff
@@ -197,7 +197,7 @@ private void initializeFeatureSources( final int featureQueryLookahead, final Co
      * Note: package-visible to enable access from the core walker classes
      * (but not actual tools, so it's not protected).
      */
-    void addToFeatureSources(final int featureQueryLookahead, final FeatureInput<? extends Feature> featureInput, final Class<? extends Feature> featureType, final int cloudPrefetchBuffer, final int cloudIndexPrefetchBuffer, final Path reference) {
+    protected void addToFeatureSources(final int featureQueryLookahead, final FeatureInput<? extends Feature> featureInput, final Class<? extends Feature> featureType, final int cloudPrefetchBuffer, final int cloudIndexPrefetchBuffer, final Path reference) {
```

Comment:
Same comment here -- if `injectFeatureDataSourcesAfterInitialization()` is moved up into `GATKTool`, this can remain package-private.

Reply:
Sounds good.  Fixed!